projects/Buttons_and_ADC/src/main.c

```diff
@@ -0,0 +1,69 @@
+#include "adc.h"
+#include "gpio.h"
+#include "gpio_it.h"
+#include "interrupt.h"
+#include "log.h"
+#include "wait.h"
+
+static void prv_button_interrupt_handler(const GpioAddress *adc_address, void *context) {
+  GpioAddress *adc_position = context;
+  uint16_t adc_data = 0;
+  StatusCode adc_converted_data = adc_read_converted_pin(*adc_position, &adc_data);
+  if (adc_converted_data == STATUS_CODE_OK) {
+    LOG_DEBUG("ADC reading is: %d\n", adc_data);
+  } 
+  else {
```

Comment:
The else should be placed beside the end of the } as shown in our coding standards
https://uwmidsun.atlassian.net/wiki/spaces/ELEC/pages/9306218/Coding+Standards
![image](https://user-images.githubusercontent.com/37023871/131696477-7e7a5fab-81dd-4b3f-9b00-7d38e119fae9.png)